docs/source/faq.rst

```diff
@@ -373,6 +373,8 @@ Here's an example:
     study = optuna.create_study()
     study.optimize(objective, n_trials=100, callbacks=[logging_callback])
 
+Note that this callback would not work as expected when you try to optimize a objective function with `n_jobs>1`.
```

Comment:
Not sure whether I should put a reason in detail here. 💭

Reply:
We could elaborate a bit perhaps. Something along

```suggestion
Note that this callback may show incorrect values when you try to optimize an objective function with `n_jobs>1` (or other forms of distributed optimization) due to its reads and writes to storage which are prone to race conditions.
```

Reply:
Nit
```suggestion
Note that this callback would not work as expected when you try to optimize a objective function with ``n_jobs!=1``.
```

Reply:
Thank you @hvy @nzw0301 for the suggestions! I applied your changes in 463c890 (sorry I didn't commit your suggestions, suggestions conflicted each other to let me manual apply...🔥)